src/app/pages/BibPage.jsx

```diff
@@ -210,6 +210,7 @@ export const BibPage = (
 
         <section style={{ marginTop: '20px' }} id="items-table">
           <ItemsContainer
+            displayDateFilter={bib.hasItemDates}
```

Comment:
Does this have a default value? Would it make sense to do `bib.hasItemDates || false`? (or true?) Otherwise, if it is undefined this will still work so this isn't a problem but just curious if it does have a default value from the API.

Reply:
i think it is always returned with a defined boolean value